phpstan.neon

```diff
@@ -1,3 +1,4 @@
 parameters:
     checkMissingIterableValueType: false
     ignoreErrors:
+    checkGenericClassInNonGenericObjectType: false
```

Comment:
@mhujer this is to discuss, whether to disable this check or not :-)

Reply:
@dfridrich let's disable it